tasks/package.yml

```diff
@@ -5,12 +5,27 @@
     pkg: python-pycurl
     state: present
 
-- name: nodejs | Add the nodejs repository
-  apt_repository:
-    repo: ppa:chris-lea/node.js
+- name: node.js | get release
+  shell: lsb_release -c -s
+  register: distro
```

Comment:
Interesting, ive just seen we already have that setup, but I guess its not covering this code, hence it passing in travis..